tests/test_storage_macro.rs

```diff
@@ -32,13 +28,50 @@ async fn test_storage_macro_redis() -> Result<(), Box<dyn std::error::Error + Se
     Ok(())
 }
 
+#[tokio::test]
+async fn test_storage_macro_append(
+) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    let (_ir, _is, cl) = get_colink_user().await?;
+
+    let key_name = "storage_macro_test_append";
+    test_append(&cl, key_name, 10 as usize).await?;
+
+    Ok(())
+}
+
+#[tokio::test]
+async fn test_storage_macro_append_redis(
+) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    let (_ir, _is, cl) = get_colink_user().await?;
+
+    cl.create_entry(
+        "storage_macro_test_append_redis:redis_url",
+        b"redis://localhost",
+    )
+    .await?;
+    let key_name = "storage_macro_test_append_redis:$redis:redis_key";
+    test_append(&cl, key_name, 5e6 as usize).await?;
+
+    Ok(())
+}
+
+#[tokio::test]
+async fn test_storage_macro_append_chunk(
+) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    let (_ir, _is, cl) = get_colink_user().await?;
+
+    let key_name = "storage_macro_test_append_chunk:$chunk";
+    test_append(&cl, key_name, 5e6 as usize).await?;
+    test_append(&cl, key_name, 10 as usize).await?;
+
+    Ok(())
+}
+
 #[ignore]
 #[tokio::test]
 async fn test_storage_macro_chunk_redis(
 ) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
-    let _ir = InstantRegistry::new();
-    let is = InstantServer::new();
-    let cl = is.get_colink().switch_to_generated_user().await?;
+    let (_ir, _is, cl) = get_colink_user().await?;
```

Comment:
wait, just realized it should be named
- `test_storage_macro_chunk_in_redis` to emphasize the semantics, or 
- `test_storage_macro_redis_chunk` to emphasize the order?